src/flamegraph/attrs.rs

```diff
@@ -1,3 +1,5 @@
+use indexmap::map::Entry;
+use indexmap::IndexMap;
```

Comment:
Is there a particular reason we need `indexmap` for this?

Reply:
Ah, it's for the tests... That's annoying. I wonder if we should just use a non-random map instead? It'd be faster too. Like an `FnvHashMap` without `RandomState`.

Reply:
Yes, for tests. I'll look into `FnvHashMap`. I've never used it.

Reply:
Take a look at the leading [documentation for `HashMap`](https://doc.rust-lang.org/std/collections/struct.HashMap.html), and note how it uses a "randomly seeded" hashing algorithm that is resistant to HashDoS. We don't need that here, and are paying for it both in terms of speed and randomized ordering. The [`FnvHashMap`](https://doc.servo.org/fnv/type.FnvHashMap.html) from the [`fnv`](https://doc.servo.org/fnv/) crate does _not_ use a random seed, and uses a weaker but faster hashing function, and should serve this purpose well. It'll also not randomize the ordering of our keys. It's perhaps worth noting that if the hashes of the strings should ever change in the future for some weird reason, then the attribute order will change, but I think that's relatively unimportant.